interop-cats/jvm/src/main/scala/scalaz/zio/interop/stream/Stream.scala

```diff
@@ -0,0 +1,157 @@
+
+  def filter(pred: A => Boolean): Stream[F, A] =
+    new Stream(underlying.filter(pred))
+
+  def filterM(pred: A => F[Boolean]): Stream[F, A] =
+    new Stream(underlying.filterM(pred.andThen(liftZIO.apply)))
+
+  def filterNot(pred: A => Boolean): Stream[F, A] =
+    filter(a => !pred(a))
+
+  def foreachWhile(f: A => F[Boolean]): F[Unit] =
+    liftIO.apply(underlying.foreachWhile(f.andThen(liftZIO.apply)))
+
+  def collect[B](pf: PartialFunction[A, B]): Stream[F, B] =
+    new Stream(underlying.collect(pf))
+
+  def drop(n: Int): Stream[F, A] =
+    new Stream(underlying.drop(n))
+
+  def dropWhile(pred: A => Boolean): Stream[F, A] =
+    new Stream(underlying.dropWhile(pred))
+
+  def flatMap[B](f: A => Stream[F, B]): Stream[F, B] =
+    new Stream(underlying.flatMap(f.andThen(_.underlying)))
+
+  def foreach(f: A => F[Unit]): F[Unit] =
+    liftIO(underlying.foreach(f.andThen(liftZIO.apply)))
+
+  def forever: Stream[F, A] =
+    new Stream(underlying.forever)
+
+  def map[B](f: A => B): Stream[F, B] =
+    new Stream(underlying.map(f))
+
+  // TODO Should we expose Chunk to clients?
+  def mapConcat[B](f: A => Chunk[B]): Stream[F, B] =
+    new Stream(underlying.mapConcat(f))
+
+  def mapM[B](f: A => F[B]): Stream[F, B] =
+    new Stream(underlying.mapM(f.andThen(liftZIO.apply)))
+
+  def merge[A1 >: A](that: Stream[F, A1], capacity: Int = 1): Stream[F, A1] =
+    new Stream(underlying.merge(that.underlying, capacity))
+
+  def mergeEither[B](that: Stream[F, B], capacity: Int = 1): Stream[F, Either[A, B]] =
+    new Stream(underlying.mergeEither(that.underlying, capacity))
+
+  def mergeWith[B, C](that: Stream[F, B], capacity: Int = 1)(l: A => C, r: B => C): Stream[F, C] =
+    new Stream(underlying.mergeWith(that.underlying, capacity)(l, r))
+
+  // TODO Sink, Managed
+  def peel[A1 >: A, B](sink: ZSink[Any, Throwable, A1, A1, B]): ZManaged[Any, Throwable, (B, Stream[F, A1])] = ???
+
+  // TODO Schedule #790
+  //  Find out how to deal correctly with "with Clock"
+  def repeat(schedule: ZSchedule[Any, Unit, _]): Stream[F, A] = ???
+    // new Stream(underlying.repeat(schedule))
+
+  // TODO Schedule #790
+  //  Find out how to deal correctly with "with Clock"
+  def repeatElems[B](schedule: ZSchedule[Any, A, B]): Stream[F, A] = ???
+    // new Stream(underlying.repeatElems(schedule))
+
+  // TODO Sink
+  def run[A0, A1 >: A, B](sink: ZSink[Any, Throwable, A0, A1, B]): F[B] =
+    liftIO(underlying.run(sink))
+
+  def mapAccum[S1, B](s1: S1)(f1: (S1, A) => (S1, B)): Stream[F, B] =
+    new Stream(underlying.mapAccum(s1)(f1))
+
+  def mapAccumM[S1, B](s1: S1)(f1: (S1, A) => F[(S1, B)]): Stream[F, B] =
+    new Stream(underlying.mapAccumM(s1) { (s1, a) => liftZIO(f1(s1, a)) })
+
+  def take(n: Int): Stream[F, A] =
+    new Stream(underlying.take(n))
+
+  def takeWhile(pred: A => Boolean): Stream[F, A] =
+    new Stream(underlying.takeWhile(pred))
+
+  def collectWhile[B](pred: PartialFunction[A, B]): Stream[F, B] =
```

Comment:
While you're at it, can you arrange these methods alphabetically so it will be easier to maintain the list going forward? 🙏 😄

Reply:
Done, but not assisted by a tool. I hope I didn't miss anything :smile: